test/functional/CacheManagement/src/tests/sharedclasses/options/TestSharedCacheJvmtiAPI.java

```diff
@@ -77,7 +102,12 @@ public static void main(String args[]) {
 		    	destroySharedCache(dir, "cache1", PERSISTENT, false);
 		    	checkFileDoesNotExistForPersistentCache("cache1");
 				if (isWindows() == false) {
-			    	destroySharedCache(dir, "cache1_groupaccess", PERSISTENT, false);
+					if (dirGroupAccess == null) {
+						destroySharedCache(dir, "cache1_groupaccess", PERSISTENT, false);
+					} else {
+						destroySharedCache(dirGroupAccess, "cache1_groupaccess", PERSISTENT, false);
+						System.out.println("GroupAccess persistent is destroy ed is " + dirGroupAccess);
```

Comment:
Typo in "GroupAccess persistent is destroy ed is". You mean "Destroy persistent cache cache1_groupaccess in groupAccess directory " ? If you might want to add similar message cache2_groupaccess

Reply:
Removed as it is for debug only.